site/content/en/install/configs.md

```diff
@@ -97,7 +97,8 @@ If more than one value is set, the order is `Managed Identity` > `Service Princi
 | routeTableResourceGroup                                    | The resource group name for routeTable                                                                                                                                                                            | Default same as resourceGroup and available since v1.15.0                                                                             |
 | loadBalancerName                                           | Working together with loadBalancerResourceGroup to determine the LB name in a different resource group                                                                                                            | Since v1.18.0, default is cluster name setting on kube-controller-manager                                                             |
 | loadBalancerResourceGroup                                  | The load balancer resource group name, which is different from node resource group                                                                                                                                | Since v1.18.0, default is same as resourceGroup                                                                                       |
-| disableAvailabilitySetNodes                                | Disable supporting for AvailabilitySet virtual machines in vmss cluster. It should be only used when vmType is "vmss" and all the nodes (including master) are VMSS virtual machines                              | Since v1.18.0, default is false                                                                                                       |
+| disableAvailabilitySetNodes                                | Disable supporting for AvailabilitySet virtual machines in vmss cluster. It should be only used when vmType is "vmss" and all the nodes (including master) are VMSS Uniform virtual machines                      | Since v1.18.0, default is false      
+| enableVmssFlexNodes                                        | Enable supporting for VMSS Flex virtual machines in vmss cluster. It should be only used when vmType is "vmss"                                                                                                    | Since v1.26.0, default is false                                                                                                       |
```

Comment:
should be only used -> should only be used

Reply:
Done